templates/barcode-development.ini.erb

```diff
@@ -0,0 +1,6 @@
+# THIS FILE IS GENERATED BY PUPPET
+# DO NOT EDIT DIRECTLY, INSTEAD EDIT THE .erb template
+
+barcode_key = "<%= @barcode_key %>"
```

Comment:
it doesn't actually matter too much because configspec.ini reads them as ints so even if we have the quotes here it'll still be ok. but, never hurts to be more correct